src/components/ApplyBrandDesigner.tsx

```diff
@@ -143,6 +143,16 @@ const ApplyBrandDesigner: React.FC = () => {
               </Typography>
             </Box>
           </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```

Comment:
Not necessarily 100% true but close enough. We might do in-person team events for fun if permissible by NEU.

Reply:
Sure, let me know if you want me to rephrase it